DMTR-421.tex

```diff
@@ -42,6 +46,995 @@
+The major change that is expected to affect data products is the adoption of a modified form of \href{https://github.com/lsst/gbdes}{gbdes} for determining astrometric solutions in the pipelines. This was accompanied by a switch from using Gaia DR2 to DR3 as the astrometric reference catalog.
+However, as shown in this document, the adoption of \texttt{gbdes} did not lead to significant changes in the astrometry metrics as measured on RC2.
+As further adaptation and refinement of \texttt{gbdes} continues these metrics are likely to improve.
+
+All metrics reported here were calculated using the \href{https://github.com/lsst/faro}{faro} (\citeds{DMTN-211}) metric calculation package, which is part of the standard pipeline builds. All of the underlying algorithms to calculate metrics within \texttt{faro} are the same as they were in v25.0.0 of the Science Pipelines, so any differences between metrics from the v25 and v26 releases are due to changes in the underlying pipelines.
+
+The metric calculation pipelines from \texttt{faro} were run on the RC2 tracts to derive the photometric, astrometric, and shape metrics that are reported here. We exclude the two astrometry metrics (AM3 and AF3) that concern residuals on 200-arcminute scales, since the individual tracts of RC2 do not span large enough spatial scales to enable these measurements.
+
+For comparison, we provide the \SRD required ``design'' value of each Key Performance Metric (KPM) as defined in the Science Requirements Document \citedsp{LPM-17}. For the ellipticity correlation metrics, there are specifications only for \emph{r} and \emph{i} bands. The \emph{ugzy}-band measurements are of interest primarily for historical tracking.
+
+Some KPMs (e.g., PF1, AF1, AF2) involve thresholds that are different for ``design'', ``minimum'', and ``stretch'' specifications. Metrics in this report are all compared to the ``design'' thresholds. The assessment of these KPMs would be different if evaluated against different thresholds.
+
+%\section{Metric monitoring dashboard}\label{sec:dashboard}
+\begin{figure}[!t]
+  \centering
+  \includegraphics[width=0.6\textwidth, trim=0.0in 0.0in 0.0in 0.0in, clip]{figures/photom_metrics_v26_with_thresholds.png}
+  \caption{Photometry metrics PA1 (photometric repeatability) and PF1 (percentage of measurements exceeding the outlier threshold) measured in the $r$-band. The figure shows the values of these metrics as measured in versions 22-26 of the LSST Science Pipelines, compared against the SRD requirements (for both the ``design'' and ``minimum'' thresholds). The measured values of both metrics have improved slightly between the previous release (v25) and the current one (v26), likely due to improvements in the robustness of aperture correction measurements to outliers.}
+  \label{fig:phot_metrics}
+\end{figure}
+
```

Comment:
Yes, because they have different units (PA1 is mmag, PF1 is %), I just included the units in the legend. If that's too confusing I could make separate plots.